compiler/src/iree/compiler/Codegen/SPIRV/test/lowering_matmul_promotion.mlir

```diff
@@ -51,34 +51,49 @@ hal.executable @matmul_f32_128x256x64 {
   }
 }
 
-//       CHECK-DAG: #[[MAP:.+]] = affine_map<(d0) -> ((d0 floordiv 16) mod 2)>
-//       CHECK-DAG: #[[TRANSLATION:.+]] = #iree_codegen.translation_info<SPIRVMatmulPromoteVectorize pipeline_depth = 2>
+//       CHECK-DAG: #[[MAP:.+]] = affine_map<(d0) -> ((d0 floordiv 16) mod 3)>
```

Comment:
Would it be easy to actually add a new test to check the case with the store_stage = 0 case? That would make sure we don't omit the previous pass in checks and only have the new case covered. If it's hard to think of a matmul shape fitting into the current heuristics, I'm thinking maybe using user config `#iree_codegen.compilation_info` to force it.

Reply:
Do you mean a test for the store_stage = 1 case? This is the store_stage = 0 case, but yeah I can add back in the previous case if that's what you mean.

Reply:
Yes sorry I meant store_stage = 1 case (which was the previous revision).